src/community/jdbcstore/src/main/resources/jdbcstore.properties

```diff
@@ -44,4 +44,4 @@ pool.validationQuery=SELECT now()
 deleteDestinationOnRename=true
 
 # leave the following directories on the filesystem -- no spaces!
-ignoreDirs=data,jdbcstore,workspaces,layergroups,temp,tmp
+ignoreDirs=data,jdbcstore,workspaces,layergroups,temp,tmp,logs
```

Comment:
@NielsCharlier - To expand upon what @jodygarnett was saying, the workspaces folder primarily contains configuration (xml files) However, for each workspace, there is a `/workspaces/<workspace_name>/styles` folder which contains styles and icons for that workspace, similar to how the top-level styles folder contains global styles.

Also, the top level styles folder already contains mixed xml configuration and styles - it has all the <stylename>.xml configuration objects as well as the actual style files. If there is no way of handling mixed stuff, then how is this getting handled right now?